src/main/java/com/neuvector/Scanner.java

```diff
@@ -185,16 +175,16 @@ private static String pullDockerImage(NVScanner nvScanner) {
 
         if(!nvRegistryUser.equals("") && !nvRegistryPassword.equals("")){
             String[] cmdArgsDockerLogin = {"docker", "login", "-u", nvRegistryUser, "-p", nvRegistryPassword, nvRegistryURL};
-            errorMessage = runCMD(cmdArgsDockerLogin, log);
+            errorMessage = runCMD(cmdArgsDockerLogin, log, null);
             if(errorMessage.length() == 0){
                 String[] cmdArgsDockerPull = {"docker", "pull", getNVImagePath(nvScannerImage, nvRegistryURL)};
-                errorMessage = runCMD(cmdArgsDockerPull, log);
+                errorMessage = runCMD(cmdArgsDockerPull, log, null);
```

Comment:
We do not need the output captured for `docker pull` or `docker logout`.

Reply:
When using nexus private registry, we need to capture docker pull and logout messages to identify, correct access to nexus. So please do not remove these details, which will hinder our troubleshooting ability. 

[ERROR] Trying to pull ubx86:9001/neuvector/scanner:latest...  <===
[ERROR] Getting image source signatures
[ERROR] Copying blob sha256:42c60484feea74980cb99c601e0103f04c51738a74b90b5a5814f212ccaa0720
[ERROR] Copying blob sha256:b519e8019fe553657e5bbf45745da5161e87752fd028737440e69465bd361a3e
[ERROR] Copying blob sha256:943552830da6ede25e2f31d23a49c4d0284231733e3d889286ddf6e452a703ea
[ERROR] Copying blob sha256:6e30ef7c00ee667e62c95748d42b8d9328d28c1c30525a80792e998ea9e0f774
[ERROR] Copying blob sha256:c50c2f264f8d031d89b149c417ed558e19ee8e9c32fb6ade626a5a74db540bc1
[ERROR] Copying config sha256:a63456af2f24667d69c9c1fa5d64dc996ca15bcc153ec661d561011eba2c22c4
[ERROR] Writing manifest to image destination
[INFO] Removed login credentials for ubx86:9001  <==

Reply:
@srini-hdp they are still being printed. No changes in behaviour. We are not 'collecting' in terms of we are not   passing it back to insight scanner.